src/content-helper/editor-sidebar/smart-linking/review-modal/component-sidebar.tsx

```diff
@@ -71,29 +103,112 @@ export const ReviewModalSidebar = ( {
 		</span>
 	);
 
+	// Build the tabs array.
+	let tabs = [];
+
+	if ( outboundLinks && outboundLinks.length > 0 ) {
+		tabs.push( {
+			name: 'outbound',
+			title: __( 'Outbound', 'wp-parsely' ),
+		} );
+	}
+
+	if ( inboundLinks && inboundLinks.length > 0 ) {
+		tabs.push( {
+			name: 'inbound',
+			title: __( 'Inbound', 'wp-parsely' ),
+		} );
+	}
+
+	let initialTabName = 'outbound';
+
+	// Change the titles of the tabs to the extended titles if there are no links.
+	tabs = tabs.filter( ( tab ) => {
+		if ( tab.name === 'outbound' && inboundLinks && inboundLinks.length === 0 ) {
+			tab.title = __( 'Outbound Smart Links', 'wp-parsely' );
+			initialTabName = 'outbound';
+		}
+		if ( tab.name === 'inbound' && outboundLinks && outboundLinks.length === 0 ) {
+			tab.title = __( 'Inbound Smart Links', 'wp-parsely' );
+			initialTabName = 'inbound';
+		}
+		return tab;
+	} );
```

Comment:
Is there a short explanation about why are we doing this?

Reply:
If there is only one tab being shown, instead of showing, for example, "Inbound" in the tab text, it can show the full "Inbound Smart Links" string. After some testing, I feel that this looks considerable better when there is only a Inbound or Outbound tab, since with the shorter string, there is plenty of empty space.

Reply:
Makes sense, thanks!